rmf_demos_fleet_adapter/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rmf_demos_fleet_adapter</name>
+  <version>0.0.0</version>
+  <description>A fleet adapter for interfacing with robots with a fleet manager via REST API</description>
+  <maintainer email="yadunund@openrobotics.org">Yadunund</maintainer>
+  <author email="xiyu@openrobotics.org">Xi Yu Oh</author>
+  <license>Apache License 2.0</license>
+
+  <exec_depend>rmf_fleet_adapter_python</exec_depend>
+
+  <depend>rclpy</depend>
+  <depend>rmf_fleet_msgs</depend>
+  <depend>rmf_task_msgs</depend>
+
```

Comment:
After https://github.com/ros/rosdistro/pull/31624 is merged, should add this for rosdep installation.
```
  <depend>univorn-pip</depend>
  <depend>fastapi-pip</depend>
```
or we can create a stopgap for this, by running the legacy fleet adapter by default. If the person would like to run the new fleetadapter and manager, they just need to swich the launch file, and install `pip3 install fastapi uvicorn` manually.

Reply:
Thanks for the rosdep PR and suggestion! I've updated the launch files with the toggle for the new fleet manager and instructions in `rmf_demos_fleet_adapter`'s readme accordingly